TextileNode.js

```diff
@@ -69,8 +65,9 @@ export default {
   sharePhoto: async function (id: string, threadName: string, caption: string): Promise<string> {
     return await TextileNode.sharePhoto(id, threadName, caption)
   },
-
+  
   getPhotoBlocks: async function (offset: ?string, limit: number, threadName: string): Promise<[any]> {
+    // TODO: JSON string coming back here... parse it
```

Comment:
Figured we should move the json parsing to the RN layer in order to have to write and maintain less native code, impl coming